setup.py

```diff
@@ -0,0 +1,27 @@
+from setuptools import setup, find_packages
+
+from rna_cd import VERSION
+
+setup(
+    name="rna_cd",
+    description="RNA contamination detection",
+    version=str(VERSION),
```

Comment:
All of your versions in all of the program are strings. So you use `str(VERSION)` all the time. I advise getting rid of this `semver` dependency and just make it a string directly.